source/platforms/platform.ts

```diff
@@ -43,20 +43,10 @@ export interface Platform {
   editMainComment: (newComment: string) => Promise<any>
   /** Replace the main Danger comment */
   updateOrCreateComment: (newComment: string) => Promise<any>
+  /** Fails the current PR */
```

Comment:
Thoughts on changing this doc comment to something like "Sets the current PR's status. See the [GitHub Statuses API documentation](https://developer.github.com/v3/repos/statuses/#create-a-status)." to document that this could either set the PR status as success or failure?